video/image_writer.c

```diff
@@ -102,8 +135,15 @@ static enum AVPixelFormat replace_j_format(enum AVPixelFormat fmt)
     return fmt;
 }
 
-static bool write_lavc(struct image_writer_ctx *ctx, mp_image_t *image, FILE *fp)
+
+static bool write_lavc(struct image_writer_ctx *ctx, mp_image_t *image, const char *filename)
```

Comment:
Is it necessary to change this to `filename` instead of just passing a `FILE` pointer?

Reply:
it's because of avio and that it's easier to do with filenames.

Reply:
Ah yeah, makes sense.